<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>350</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>GeoImage Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="gbOpenSave">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>55</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1071</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="title">
        <string>Масштабирование</string>
       </property>
       <widget class="QPushButton" name="bDecrease">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bIncrease">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbScale">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>281</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Текущий масштаб: </string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="geoLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbBrights">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <widget class="QMdiArea" name="mdiArea">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lb2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="mOpen">
    <property name="title">
     <string>Изображение</string>
    </property>
    <addaction name="actionSaveRPC"/>
    <addaction name="actionAccuracy"/>
    <addaction name="actionHistogram"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Трансформация изображений</string>
    </property>
    <addaction name="action2Images"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="mOpen"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSaveRPC">
   <property name="text">
    <string>Сохранить RPC</string>
   </property>
  </action>
  <action name="actionAccuracy">
   <property name="text">
    <string>Оценка точности</string>
   </property>
  </action>
  <action name="action2Images">
   <property name="text">
    <string>Наложить 2 изображения</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="text">
    <string>Гистограмма</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить изображение</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
